_maps/configs/IndiCaravanTypeA.json

```diff
@@ -0,0 +1,26 @@
+{
+	"map_name": "Caravan-class Modular ship",
+	"map_short_name": "Caravan-class",
+	"map_path": "_maps/shuttles/shiptest/IndiCaravanTypeA.dmm",
+	"map_id": "IndiCaravanTypeA",
+	"roundstart": true,
+	"job_slots": 
+	{
+		"Captain": 
+		{
+			"outfit": "/datum/outfit/job/captain/western",
+			"officer": true,
+			"slots": 1
+		},
+		"Scientist": 1,
+		"Medical Doctor": 1,
+		"Atmospheric Technician": 1,
```

Comment:
you may want to use custom outfits and titles for these three specialist jobs, as the defaults for all three are rather NT-themed. Even if you stick to the default outfits a custom title still adds some useful flavor.